### DISCUSSION TITLE
npm install -D prettier-plugin-tailwindscss *is not working*

### DISCUSSION THREAD

**Discussion by jcodedeveloper** [tailwindlabs/tailwindcss#13440] [Help] [answered]
When I´m going to in the terminal write the code npm install -D prettier-plugin-tailwindcss it doesn't work. He doesn't organize tailwind classes for me. Please anybody have the solve?

**Comment by alex-krasikau** [comment, marked as answer]
Hey @jcodedeveloper!

To make the Prettier Tailwind plugin work in your project, follow these steps:

1. Add the Tailwind Prettier plugin to your prettier configuration:
```javascript
// prettier.config.js
module.exports = {
  plugins: ['prettier-plugin-tailwindcss'],
}
```

2. If you're using VSCode, install the `Prettier - Code formatter` plugin.
<img width="330" alt="CleanShot 2024-04-04 at 12 39 49@2x" src="https://github.com/tailwindlabs/tailwindcss/assets/97882909/820c7037-6821-4e34-89cf-e81645f8fa7f">


3. Enable format on save by adding this to your VSCode `settings.json`:
```json
"editor.defaultFormatter": "esbenp.prettier-vscode",
"editor.formatOnSave": true
```

**Comment by jcodedeveloper** [comment]
thanks for giving me the help
